src/qw/md.py

```diff
@@ -0,0 +1,16 @@
+"""Prototyping of extracting structured information from markdown."""
+
+from markdown import markdown
+from parsel import Selector
+
+
+def text_under_heading(text: str, heading: str) -> str:
+    """Extract text elements under a specific heading."""
+    html = markdown(text)
+    selector = Selector(html)
+    header = selector.xpath('//h3[contains(text(), "What happened?")]')
```

Comment:
I assume you mean `f'//h3[text()= "{heading}"]'` not `'//h3[contains(text(), "What happened?")]'`